### ISSUE TITLE
Events Section Is A Mess

### ISSUE THREAD

**Issue by sunscreem** [AberdeenPHP/AberdeenPHP-Website#42]
The way the events are displayed on the events page is a bit meh.

Rowan suggest a month view calendar type idea? 

At the very least we could do with reducing the amount of near "duplicates" (57North - looking at you!) and reduce the amount of text on the page by collapsing things [behind an accordion](http://getbootstrap.com/javascript/#collapse) or similar.

If anyone fancies making this, please jump in.

**Comment by AndyGaskell** [comment]
I think the format is a good thing.  I know what you mean, a month view calender is the norm, but that'd look a bit rubbish with only or two entries per month.  I was thinking the vibe of the site is kind of chatty and not overly complex, so the text format fits this well.  

The user story for this page is, I think, something like "When is the next meet-up and what's it about?" and to my mind https://aberdeenphp.co.uk/events is pretty good for that.

**Comment by evocate** [comment]
I agree with @sunscreem.  It's too much descriptive text to find the important information (what, where, when) quickly.  Agree also with @AndyGaskell with the point that an almost empty calendar would look poor.

I have what I think may be a simple solution.  That for now, we have an overview of upcoming events at the top of the page.  On desktop this could show the next 4 and on mobile 3 (2 maybe if things get a bit tight) events.  These would be links to anchor points on the page.  Because we are bias AberdeenPHP would always show in this list it doesn't matter if there was 10 events in between now and the next AberdeenPHP Meetup.

I have made a mock up on how this could potentially look, if there is no big protests I will crack on with making it.

![image](https://user-images.githubusercontent.com/13115839/31577573-fc122a9c-b108-11e7-9fd0-4b872af2f330.png)

**Comment by sunscreem** [comment]
Richard - thank you fella.

Would you be thinking these cards would replace the data below or in addition too it?

Trying to put myself in the shoes of a new person looking at that mockup and I'm wondering why some events are in cards and others in text.  Maybe that section needs a label, or perhaps the whole lot needs to be cards (which some way of expanding them to show the detail?) What'dya fink?

**Comment by evocate** [comment]
It would be in addition too.  The purpose of these cards would be just a visual overview of what will be the soonest events.  Each card would link to the relevant section / text below.

It's a more agile solution, because a full solution will take more time.  In the future we could turn potentially turn this into a carousel and / or have a link to a full calendar overview.

**Comment by sunscreem** [comment]
Gotcha. 

I think you should fire on in that case.

I want to re-write the code that brings the events in so I'll wait for you to do that front end stuff first and then I'll do my bit.

**Comment by evocate** [comment]
Not really relevant in this case but in future how limited I here of using a JS framework?  Something like vue.js?

**Comment by sunscreem** [comment]
No limits at all Sir. So long as others understand what's going on and we stick to well documented stuff I would say anything goes.
